[2025-10-17T20:53:44.279+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_run_pipeline.dbt_run manual__2025-10-17T20:53:32.974715+00:00 [queued]>
[2025-10-17T20:53:44.295+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_run_pipeline.dbt_run manual__2025-10-17T20:53:32.974715+00:00 [queued]>
[2025-10-17T20:53:44.295+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-10-17T20:53:44.329+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): dbt_run> on 2025-10-17 20:53:32.974715+00:00
[2025-10-17T20:53:44.341+0000] {standard_task_runner.py:57} INFO - Started process 191 to run task
[2025-10-17T20:53:44.346+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dbt_run_pipeline', 'dbt_run', 'manual__2025-10-17T20:53:32.974715+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmpe_no2mrt']
[2025-10-17T20:53:44.350+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask dbt_run
[2025-10-17T20:53:44.373+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-17T20:53:44.435+0000] {task_command.py:415} INFO - Running <TaskInstance: dbt_run_pipeline.dbt_run manual__2025-10-17T20:53:32.974715+00:00 [running]> on host eb8a9840e92a
[2025-10-17T20:53:44.576+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_run_pipeline' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2025-10-17T20:53:32.974715+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-17T20:53:32.974715+00:00'
[2025-10-17T20:53:44.577+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-17T20:53:44.579+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /opt/***/dbt && dbt run --profiles-dir /opt/***/dbt']
[2025-10-17T20:53:44.589+0000] {subprocess.py:86} INFO - Output:
[2025-10-17T20:53:47.770+0000] {subprocess.py:93} INFO - [0m20:53:47  Running with dbt=1.7.0
[2025-10-17T20:53:48.111+0000] {subprocess.py:93} INFO - [0m20:53:48  Registered adapter: postgres=1.7.0
[2025-10-17T20:53:48.182+0000] {subprocess.py:93} INFO - [0m20:53:48  Found 1 model, 1 seed, 0 sources, 0 exposures, 0 metrics, 401 macros, 0 groups, 0 semantic models
[2025-10-17T20:53:48.185+0000] {subprocess.py:93} INFO - [0m20:53:48
[2025-10-17T20:53:48.330+0000] {subprocess.py:93} INFO - [0m20:53:48  Concurrency: 1 threads (target='dev')
[2025-10-17T20:53:48.331+0000] {subprocess.py:93} INFO - [0m20:53:48
[2025-10-17T20:53:48.341+0000] {subprocess.py:93} INFO - [0m20:53:48  1 of 1 START sql table model public_public.transform_data ...................... [RUN]
[2025-10-17T20:53:48.481+0000] {subprocess.py:93} INFO - [0m20:53:48  1 of 1 OK created sql table model public_public.transform_data ................. [[32mSELECT 3[0m in 0.14s]
[2025-10-17T20:53:48.499+0000] {subprocess.py:93} INFO - [0m20:53:48
[2025-10-17T20:53:48.500+0000] {subprocess.py:93} INFO - [0m20:53:48  Finished running 1 table model in 0 hours 0 minutes and 0.31 seconds (0.31s).
[2025-10-17T20:53:48.515+0000] {subprocess.py:93} INFO - [0m20:53:48
[2025-10-17T20:53:48.516+0000] {subprocess.py:93} INFO - [0m20:53:48  [32mCompleted successfully[0m
[2025-10-17T20:53:48.517+0000] {subprocess.py:93} INFO - [0m20:53:48
[2025-10-17T20:53:48.518+0000] {subprocess.py:93} INFO - [0m20:53:48  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-10-17T20:53:49.429+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-17T20:53:49.460+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=dbt_run_pipeline, task_id=dbt_run, execution_date=20251017T205332, start_date=20251017T205344, end_date=20251017T205349
[2025-10-17T20:53:49.493+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-10-17T20:53:49.515+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
